helm/observability-bundle/ksm-configurations/upbound_crd_v1beta1.yaml

```diff
@@ -0,0 +1,151 @@
+rbac:
+  - apiGroups:
+      - iam.aws.upbound.io
+    resources:
+      - roles
+    verbs: ["list", "watch"]
+  - apiGroups:
+      - sqs.aws.upbound.io
+    resources:
+      - queues
+      - queuepolicies
+    verbs: ["list", "watch"]
+  - apiGroups:
+      - cloudwatchevents.aws.upbound.io
+    resources:
+      - rules
+      - targets
+    verbs: ["list", "watch"]
+  - apiGroups:
+      - ec2.aws.upbound.io
+    resources:
+      - securitygroups
+    verbs: ["list", "watch"]
+
+resources:
+  - groupVersionKind:
+      group: iam.aws.upbound.io
+      kind: Role
+      version: v1beta1
+    metricNamePrefix: iam_aws_upbound_role
+    labelsFromPath:
+      name:
+        - metadata
+        - name
+      uid:
+        - metadata
+        - uid
```

Comment:
You sure we want the uid? For cardinality reasons I mean?

Reply:
Removed